replicate.Rmd

```diff
@@ -0,0 +1,1684 @@
+  filter(term == "logses") %>%
+  mutate(beta = round(estimate, digit = 2),
+         p = round(p.value, digit = 4)) %>%
+  select(term, beta, p)
+
+logit2logses <- glm(tradeoff ~ logses, data = clean_1a, family = "binomial")
+
+print(wald.test(b = coef(logit2logses), Sigma = vcov(logit2logses), Terms = 2)$result$chi2[1], digit = 3)
+```
+
+```{r echo = FALSE}
+figure1alow <- clean_1a %>%
+  filter(income2 <= 35000) %>%
+  mutate(total.n = n(), total.tradeoff = sum(tradeoff), prop.tradeoff = sum(tradeoff)/n(), total.location = sum(location), prop.location = sum(location)/n(), SE.tradeoff = sqrt(prop.tradeoff * (1 - prop.tradeoff)/total.n), CI.tradeoff = qnorm(.975) * SE.tradeoff, SE.location = sqrt(prop.location * (1 - prop.location)/total.n), CI.location = qnorm(.975) * SE.location) %>%
+  distinct(prop.tradeoff, prop.location, CI.tradeoff, CI.location)
+#computes new variables for proportion of answers, standard errors, and 95% confidence intervals for lower income group
+
+
+figure1ahigh <- clean_1a %>%
+    filter(income2 > 35000) %>%
+  mutate(total.n = n(), total.tradeoff = sum(tradeoff), prop.tradeoff = sum(tradeoff)/n(), total.location = sum(location), prop.location = sum(location)/n(), SE.tradeoff = sqrt(prop.tradeoff * (1 - prop.tradeoff)/total.n), CI.tradeoff = qnorm(.975) * SE.tradeoff,  SE.location = sqrt(prop.location * (1 - prop.location)/total.n), CI.location = qnorm(.975) * SE.location) %>%
+  distinct(prop.tradeoff, prop.location, CI.tradeoff, CI.location)
+#computes new variables for proportion of answers, standard errors, and 95% confidence intervals for higher income group
+
+title = factor(c("trade-offs", "location"), levels = c("trade-offs", "location"))
+values = c(figure1alow$prop.tradeoff, figure1alow$prop.location, figure1ahigh$prop.tradeoff, figure1ahigh$prop.location)
+income = c("lower income", "lower income", "higher income", "higher income")
+CIs = c(figure1alow$CI.tradeoff, figure1alow$CI.location, figure1ahigh$CI.tradeoff, figure1ahigh$CI.location)
+fig1a.table <- data.frame(title, values, income, CIs)
+income.factor <- factor(fig1a.table$income, levels = c("lower income", "higher income"))
+#figure 1a data frame (+ income.factor reverses the default position dodge of the bars)
+
+figure1a <- fig1a.table %>%
+  ggplot(aes(x = title, y = values, fill = income.factor)) + geom_bar(stat = "identity", position = position_dodge(), colour = "black") + ylab("Proportion Endorsing Reason") + xlab(NULL) + geom_errorbar(aes(ymin = values - CIs, ymax = values + CIs), width = 0.2, position = position_dodge(0.9)) + theme_bw() + scale_fill_grey(start = .3, end = .85) + ggtitle("Figure 1a - the proportion of lower- and higher-income participants\nciting trade-off and location considerations in Study 1a\n(error bars represent 95% confidence interval)") + theme(panel.border = element_blank(), panel.grid.major = element_blank(), panel.grid.minor = element_blank(), axis.line = element_line(colour = "black"))
+#produces figure 1a
```

Comment:
Agree with Matt - if I was attempting to replicate or borrow from this code, I'd have to spend time breaking it up into pieces.